database/sql/schema_version_29.sql

```diff
@@ -0,0 +1 @@
+alter table users add column custom_css text default '';
```

Comment:
Another possibility to avoid altering the schema is to use the extra field in users table. It's a map/hstore.

Reply:
Ooh cool, I'll give this a go, thank you!